Library/Homebrew/cmd/install.rb

```diff
@@ -241,10 +244,15 @@ def install
       return if formulae.empty?
       Install.perform_preinstall_checks
 
+      build_times = []
       formulae.each do |f|
         Migrator.migrate_if_needed(f)
+        t0 = Time.now
```

Comment:
Technically this won't be a "build time" but a "installation time" as it includes e.g. pouring, post-install.

Reply:
Changed the terminology to "install times".